apps/api/src/server/app/validation/validation.controller.js

```diff
@@ -36,4 +37,14 @@ const getValidation = function (request, response) {
 	response.send(appealsList);
 };
 
-export { getValidation, getAppealReview };
+const updateValidation = function (request, response) {
+	console.log(request.text);
+	const errors = validationResult(request);
+	if (!errors.isEmpty()) {
+		console.log(errors);
```

Comment:
In the future, I think we can remove these `console.log`

Reply:
will do in the next PR